libs/npm/core/src/components/flex/row/styled/Row.tsx

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components';
 import { IRowProps } from '..';
 
 export const Row = styled.div<IRowProps>`
+  width: ${(props) => props.width ?? '100%'};
```

Comment:
Not sure if this is good or not.  I believe it does make alignment within the row an issue...  But not certain about this.